migrations/sql/V2019.03.25.11.06__add_hsrc_code.sql

```diff
@@ -0,0 +1,1051 @@
+CREATE TABLE IF NOT EXISTS article_act_code
+(
+    article_act_code character varying(5)                                NOT NULL,
+    description      character varying(100)                              NOT NULL,
+    effective_date   date                     DEFAULT now()              NOT NULL,
+    expiry_date      date                     DEFAULT '9999-12-31'::date NOT NULL,
+    update_timestamp timestamp with time zone DEFAULT now()              NOT NULL
+);
+
+COMMENT ON TABLE article_act_code IS 'Contains a list of compliance sections as defined in a Legislative Act or Regulation; i.e. HSRC 1.2.2';
+
+ALTER TABLE ONLY article_act_code
+    ADD CONSTRAINT article_act_code_unique UNIQUE (article_act_code);
+
+INSERT INTO article_act_code
+(
+    article_act_code,
+    description,
+    effective_date,
+    expiry_date,
+    update_timestamp
+)
+VALUES
+(
+    'HSRCM',
+    'Health, Safety and Reclamation Code for Mines in BC (MA)',
+    '1970-01-01',
+    '9999-12-31',
+    '2018-05-15'
+),
+(
+    'MA',
+    'Mines Act (293/1996)',
+    '1970-01-01',
+    '9999-12-31',
+    '2018-05-15'
+);
+
+CREATE TABLE IF NOT EXISTS compliance_article
+(
+    article_act_code  character varying(5)                                NOT NULL,
+    section           integer                                             NOT NULL,
+    sub_section       integer                                                     ,
+    paragraph         integer                                                     ,
+    sub_paragraph     integer                                                     ,
+    description       character varying(100)                              NOT NULL,
+    effective_date    date                     DEFAULT now()              NOT NULL,
+    expiry_date       date                     DEFAULT '9999-12-31'::date NOT NULL,
+    article_type_code character varying(1)                                NOT NULL
```

Comment:
We probably don't need this column... my apologies for adding it to the spreadsheet. This would be another code table type (A = Act and R = Regulation). Probably doesn't provide any business value at this point?